server/ip_hostname_mapping.py

```diff
@@ -0,0 +1,21 @@
+import json
+import socket
```

Comment:
Shouldn't we add `socket` to the docker requirement ?

Reply:
This should be lunch by a cron jon outside docker. Moreover socket is pre-installed :)

Reply:
Indeed ! Double indeed :)

However, if this job is not used by the wodker/webserver, shoulnd't it be in pyroengine/pi_utils ?

Reply:
ok, I've made the change :)

Reply:
Perfect, thanks a lot :)